src/constants/projects/competences.js

```diff
@@ -1,26 +1,26 @@
 export default [
   {
-    name: 'creativity',
-    checked: true,
+    id: 0,
```

Comment:
This is a JS file, not JSON. could've simply used 
```js
['creativity', 'critical thinking', '...'].map((label, id) => ({label, id}));
```
😁

Reply:
This comment has just changed my mind. Will add a task for this to use it in all -group components later